cmake/build_configurations/mysql_release.cmake

```diff
@@ -89,6 +89,11 @@ IF(WIN32)
   SET(INSTALL_MYSQLTESTDIR "" CACHE STRING "")
   SET(INSTALL_SQLBENCHDIR  "" CACHE STRING "")
   SET(INSTALL_SUPPORTFILESDIR ""  CACHE STRING "")
+ELSEIF(CMAKE_SYSTEM_NAME MATCHES "AIX")
+  # AIX freesource is RPM, but different than Linux RPM
+  SET(WITH_SSL system CACHE STRING "")
+  SET(WITH_ZLIB system CACHE STRING "")
+  SET(WITH_LIBARCHIVE STATIC CACHE STRING "")
```

Comment:
we do not use libarchive anymore (we might have, long ago , for a short time). There is no  WITH_LIBARCHIVE in the codebase. So this line should be removed.

Reply:
Done.